app/assets/stylesheets/errors.sass

```diff
@@ -0,0 +1,20 @@
+body#errors
+  background-color: #F4F4F4
+  color: #2E2F30
+  text-align: center
+  font-family: arial, sans-serif
+  margin: 0
+
+h1.errors
```

Comment:
These need to be indented so they are inside `body#errors`. Then you don't have to add `.errors` class to `h1` and `p`.